### DISCUSSION TITLE
version 2.0.4 not support group_concat function

### DISCUSSION THREAD

**Discussion by Tsao-zy** [sqlalchemy/sqlalchemy#10546] [Usage Questions] [answered]
in version 2.0.4, neither `func.group_concat` or `func.aggregate_string` can not import, Which function can achieve the effect of `group_concat` in mysql？

**Comment by zzzeek** [comment, marked as answer]
the `func` construct generates any function you give it, so `func.group_concat()` is available in any SQLAlchemy version.  "aggregate_strings()" is just a helper function for dialect-agnostic use, but if you are on mysql only, you don't need it. 

```py
>>> import sqlalchemy
>>> sqlalchemy.__version__
'2.0.4'
>>> from sqlalchemy import select, column, func
>>> stmt = select(column('q'), func.group_concat(column('y'))
... )
>>> print(stmt)
SELECT q, group_concat(y) AS group_concat_1

```
